extensions/binding/org.eclipse.smarthome.binding.lifx/README.md

```diff
@@ -79,6 +79,9 @@ The **porch** light is a LIFX+ BR30 that has a *colorirlight* thing type which s
 
 Finally, **kitchen** is a White 800 (Low Voltage) light that has a *whitelight* thing type which supports *brightness* and *temperature* channels.
 
+It is possible to link *Switch* and *Dimmer* items to the *color* channel. However it is not recommended.
```

Comment:
> However it is not recommended.

Well, you can do so, if this is all what you need. My comment was that you should not define such items ADDITIONALLY to a Color item as this merely duplicates the functionality. Linking a color channel to a Switch or Dimmer item is nothing wrong in general.